types/node/index.d.ts

```diff
@@ -2354,7 +2354,7 @@ declare module "net" {
         unref(): void;
         ref(): void;
 
-        remoteAddress: string;
```

Comment:
@kazuyamamoto Since we've reached consensus here, pls create another commit that make both remotePort and remoteFamily optional. thanks!